spec/aws_loader_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe AwsLoader do
+
+  let (:mod_test) { ModuleTest.new }
+
+  context 'aws.yml file' do    
+    context '#configure_aws_file' do
+      it 'should return false for aws.yml file check and call #configure_aws_file and create a new .yml file' do
+        stub_const('AwsLoader::AWS_PATH', nil)
+        allow(File).to receive(:exists?).and_return(false)
+        expect(STDOUT).to receive(:puts).and_return('No `aws.yml` file present!')
+        expect(STDOUT).to receive(:puts).and_return('Created file and set the appropriate values!')
```

Comment:
The above 2 expectations are syntactically incorrect. Should be 
`expect(STDOUT).to receive(:puts).with('No `aws.yml` file present!')` - we are expecting `puts` with a specific string.
Same as below test